app/controllers/api/v1/rooms_controller.rb

```diff
@@ -35,6 +35,14 @@ def popular
                limit(max_num)
   end
 
+  def recommend
+    max_num = 10
+
+    @rooms = Room.where(
+      id: RecommendRoom.order("RAND()").limit(max_num).pluck(:room_id),
```

Comment:
```suggestion
      id: RecommendRoom.order("RAND()").limit(max_num).select(:room_id),
```
`select`にしたら1クエリになります